src/nunit-gui/Presenters/MainPresenter.cs

```diff
@@ -168,23 +195,50 @@ private void MainForm_Closing(object sender, FormClosingEventArgs ea)
             }
         }
 
-        #region Command Handlers
+        private void SelectedRuntime_SelectionChanged()
+        {
+            ChangePackageSetting(EnginePackageSettings.RuntimeFramework, _view.SelectedRuntime.SelectedItem);
+        }
 
-        private void OnOpenProjectCommand()
+        private void ProcessModel_SelectionChanged()
         {
-            string[] files = _view.DialogManager.GetFilesToOpen();
-            if (files.Length > 0)
-                _model.LoadTests(files);
+            ChangePackageSetting(EnginePackageSettings.ProcessModel, _view.ProcessModel.SelectedItem);
         }
 
-        private void OnReloadTestsCommand()
+        private void DomainUsage_SelectionChanged()
         {
-           _model.ReloadTests();
+            ChangePackageSetting(EnginePackageSettings.DomainUsage, _view.DomainUsage.SelectedItem);
         }
 
-        private void OnReloadTestsCommand(string runtime)
+        private void LoadAsX86_CheckedChanged()
         {
-            _model.ReloadTests(runtime);
+            var key = EnginePackageSettings.RunAsX86;
+            if (_view.RunAsX86.Checked)
+                ChangePackageSetting(key, true);
+            else
+                ChangePackageSetting(key, null);
+        }
+
+        private void ChangePackageSetting(string key, object setting)
+        {
+            if (setting == null || setting is string && (string)setting == "DEFAULT")
+                _model.PackageSettings.Remove(key);
+            else
+                _model.PackageSettings[key] = setting;
+
+            string message = string.Format("New {0} setting will not take effect until you reload.\r\n\r\n\t\tReload Now?", key);
```

Comment:
Not a fan of string interpolation. 😞 We generally do not use . var for ints, bools, strings or other basic Types with short names.